### PR TITLE
Add ts sdk tests docs

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Description
Following a user feedback on not having docs around the TS SDK integration tests, adding a page covers the overall SDK tests and how one can build their own integration tests against a local node
### Checklist

- [] Do all Lints pass?
- [] Have you ran `pnpm spellcheck`?
- [] Have you ran `pnpm fmt`?
- [] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+await localNode.run();
+// Write some tests
+test("test my app", async () => {
+  const aptosConfig = new AptosConfig({network:Network.LOCAL})
+  const aptos = new Aptos(aptosConfig)
+  // rest of test.....
+}
+// Stop the local node
+localNode.stop();
+```
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:70]: Do we have to mention to download `aptos` CLI first?

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```ts
+test("transaction submission", async () => {
+  const transaction = await aptos.transaction.build.simple({
+    sender: sender.accountAddress,
+    data: {
+      function: `0x1::aptos_account::transfer`,
+      functionArguments: [receiver.accountAddress, 1],
+    },
+  });
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:38]: I'd just put a single comment line here, "replace with your application's transaction"

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```ts
+test("account address is valid", () => {
+  const { valid } = AccountAddress.isValid({
+    input: "0x1",
+  });
+  expect(valid).toBe(true);
+});
+```
+
+Can check [here](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests/unit) all the SDK unit tests
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:22]:
```suggestion
Can check [here](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests/unit) for the SDK unit tests
```

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Tests"
+---
+
+# SDK Tests
+
+The SDK uses two types of tests, `e2e` and `unit` tests, located under the [tests](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests) folder.
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:7]:
```suggestion
The SDK uses two types of tests, `End-to-End (E2E)` and `unit` tests, located under the [tests](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests) folder.
```

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  });
+  expect(valid).toBe(true);
+});
+```
+
+Can check [here](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests/unit) all the SDK unit tests
+
+## E2E Tests
+
+End-to-end tests are meant to test the end-to-end operations starting from the SDK methods to the interaction with the REST/Indexer API and a smart contract and up to the blockchain level.
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:26]:
```suggestion
End-to-end tests are meant to test the end-to-end operations starting from the SDK methods through to the blockchain.
```

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+For example, to test if a transaction has been submitted, we start with building the transaction payload the SDK expects, post the submit request to the REST API, and fetch the transaction data to make sure it has been fully submitted to the blockchain.
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:28]:
```suggestion
For example, to test if a transaction has been submitted, we start with building a transaction payload that the SDK expects, submit the request to the REST API, and fetch the transaction data to make sure it has been fully committed to the blockchain.
```

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  });
+  expect(response.success).toBe(true);
+});
+```
+
+Can check [here](https://github.com/aptos-labs/aptos-ts-sdk/tree/main/tests/e2e) all the SDK e2e tests
+
+## Integration Tests
+
+The SDK provides an easy way to run integration tests by spinning up a local node and run tests against it. For example, one can build their integration tests against a local node with the sdk like that:
```
Comment (gregnazario) [inline apps/docusaurus/docs/sdks/ts-sdk/tests.md:54]:
```suggestion
The SDK provides an easy way to run integration tests by spinning up a local node and running tests against it. For example, one can build their integration tests against a local node with the SDK like below:
```

**Review by gregnazario — APPROVED** [review]
Amazing!  Maybe we rename the page to `Testing` over `Tests`, but a much needed guide!

File: apps/docusaurus/docs/sdks/ts-sdk/tests.md (hunk 1 above)
Comment (0xmaayan) [reply apps/docusaurus/docs/sdks/ts-sdk/tests.md:70]: nope, comes with the sdk

**Pull request merged**